tests/Unit/Elliptic/DiscontinuousGalerkin/Test_DgOperator.cpp

```diff
@@ -755,6 +756,134 @@ SPECTRE_TEST_CASE("Unit.Elliptic.DG.Operator", "[Unit][Elliptic]") {
       }
     }
   }
+  {
+    INFO("2D with h-nonconforming boundary");
+    using system =
+        Poisson::FirstOrderSystem<2, Poisson::Geometry::FlatCartesian>;
+    Poisson::Solutions::ProductOfSinusoids<2> analytic_solution{{{M_PI, M_PI}}};
+    using boundary_condition_registrars =
+        typename system::boundary_conditions_base::registrars;
+    using AnalyticSolutionBoundaryCondition =
+        typename elliptic::BoundaryConditions::Registrars::AnalyticSolution<
+            system>::template f<boundary_condition_registrars>;
+    Parallel::register_derived_classes_with_charm<
+        typename system::boundary_conditions_base>();
+    {
+      INFO("Regression tests");
+      // Domain decomposition:
+      // ^ eta
+      // +-+-+> xi
+      // | | |
+      // +-+ |
+      // | | |
+      // +-+-+
+      const domain::creators::AlignedLattice<2> domain_creator{
+          // Start with 2 unrefined blocks
+          {{{0., 0.25, 0.5}, {0., 0.5}}},
+          {{0, 0}},
+          {{3, 2}},
+          // Refine once in eta in left block in sketch above
+          {{{{0, 0}}, {{1, 1}}, {{0, 1}}}},
+          {},
+          {},
+          std::make_unique<AnalyticSolutionBoundaryCondition>(
+              elliptic::BoundaryConditionType::Dirichlet)};
+      const ElementId<2> lowerleft_id{0, {{{0, 0}, {1, 0}}}};
+      const ElementId<2> upperleft_id{0, {{{0, 0}, {1, 1}}}};
+      const ElementId<2> right_id{1, {{{0, 0}, {0, 0}}}};
+      using Vars = Variables<tmpl::list<Var<Poisson::Tags::Field>>>;
+      using PrimalFluxes = Variables<tmpl::list<Var<::Tags::Flux<
+          Poisson::Tags::Field, tmpl::size_t<2>, Frame::Inertial>>>>;
+      using OperatorVars =
+          Variables<tmpl::list<DgOperatorAppliedTo<Var<Poisson::Tags::Field>>>>;
+      Vars vars_rnd_lowerleft{6};
+      get(get<Var<Poisson::Tags::Field>>(vars_rnd_lowerleft)) = DataVector{
```

Comment:
Would it be useful to add a comment where these numbers came from? Eg did you compute them externally or use them from SpECTRE and this only serves as a way to tell that the code has changed?

Reply:
These particular numbers I computed on a branch from June that I used for the tests in the paper. The other numbers in the file were computed from a Python implementation at the time. Now they serve as a regression test, to make sure the implementation doesn't change. There are a few notes at the beginning of the test case in this file, I think those are sufficient.